pkg/spiffe/spiffe.go

```diff
@@ -241,6 +241,7 @@ func RetrieveSpiffeBundleRootCerts(config map[string]string, caCertPool *x509.Ce
 			} else if resp.StatusCode != http.StatusOK {
 				b := make([]byte, 1024)
 				n, _ := resp.Body.Read(b)
+				_ = resp.Body.Close()
```

Comment:
are we cllsing twice now? L260

Reply:
This still applies, I am not sure why this was resolved?

Reply:
@jiuker the Body closed is handled by the defer in line 259.
Why do you think we should close it in this block as well?
Please close the PR if you think it's redundant.